x-pack/metricbeat/module/enterprisesearch/docker-compose.yml

```diff
@@ -15,8 +15,8 @@ services:
       - "elasticsearch.password=changeme"
       - "elasticsearch.host=http://elasticsearch:9200"
       - "allow_es_settings_modification=true"
-      - "ent_search.auth.native1.source=elasticsearch-native"
       - "secret_management.encryption_keys=[4a2cd3f81d39bf28738c10db0ca782095ffac07279561809eecc722e0c20eb09]"
+      - "kibana.host=http://localhost:5601"
```

Comment:
This is needed to make Enterprise Search start, we don't actually need Kibana there